### PR TITLE
Created the SigmaDeltaFilter class

### PR DESCRIPTION
**Pull request by irish3725**
#### Changes:
The class has a constructor that initializes the List and a method called Filter that takes a BufferedImage and returns a BufferedImage. It doesn't do anything with it yet.
Most of the time I spent tonight was making sure that my computer would play nicely with java, IntelliJ, gradle, and git. I eventually got it all to work though so tomorrow i will work on actually getting something useful inside the SigmaDeltaFilter
The fist pull request I submitted had the class saved in a weird place so I fixed that.

Please Review: @sdilts @sagesmith-wf

### CODE REVIEW

**Comment by sdilts** [comment]
You need to add testing functions for it to be happy. See [https://github.com/codecov/support/wiki/Patch-Status](https://github.com/codecov/support/wiki/Patch-Status)

**Comment by sagersmith8** [comment]
Add some unit tests dude... Thanks

File: src/main/java/csci432/SigmaDeltaFilter.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    public SigmaDeltaFilter() {
+        this.imageList = new ArrayList<BufferedImage>();
+    }
+
+    public BufferedImage Filter(BufferedImage b) {
+        return b;
+    }
+
+}
```
Comment (sagersmith8) [inline src/main/java/csci432/SigmaDeltaFilter.java]: Add a newline for github

File: src/main/java/csci432/SigmaDeltaFilter.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+public class SigmaDeltaFilter {
+
+    List<BufferedImage> imageList;
+
+    public SigmaDeltaFilter() {
+        this.imageList = new ArrayList<BufferedImage>();
+    }
+
+    public BufferedImage Filter(BufferedImage b) {
```
Comment (sagersmith8) [inline src/main/java/csci432/SigmaDeltaFilter.java]: Add some java docs

Comment (sagersmith8) [inline src/main/java/csci432/SigmaDeltaFilter.java]: Use java naming conventions and rename this `filter`

File: src/main/java/csci432/SigmaDeltaFilter.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        return b;
+    }
+
```
Comment (sagersmith8) [inline src/main/java/csci432/SigmaDeltaFilter.java:46]: Remove line please

File: src/main/java/csci432/SigmaDeltaFilter.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package csci432;
+
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SigmaDeltaFilter {
+
```
Comment (sagersmith8) [inline src/main/java/csci432/SigmaDeltaFilter.java:6]: Remove this line please

**Comment by sagersmith8** [comment]
You need to pull upstream master I think

**Pull request merged**